insights/parsers/ip.py

```diff
@@ -325,7 +325,6 @@ class RouteDevices(CommandParser):
     # Why have types we ignore?
     IGNORE_TYPES = set(["broadcast",
                         "throw",
-                        "local",
```

Comment:
I think better move the "local" to the `SAVED_TYPES`

Reply:
Done